vm-setup/roles/v1aX_integration_test/filter_plugins/status.py

```diff
@@ -0,0 +1,70 @@
+""" filtering k8s_info resources """
+
+from ansible.errors import AnsibleFilterError
+from ansible.utils.display import Display
+
+display = Display()
+
+
+def msg(key, resources):
+    return "could not find %r key in %r" % (key, resources)
+
+
+def filter_phase(resources, phase, skip_missing=True):
+    """return resources in a defined phase"""
+    filtered = []
+    for r in resources:
+        try:
+            if r["status"]["phase"].lower() == phase:
+                filtered.append(r)
+        except KeyError:
+            if skip_missing:
+                display.warning(msg("['status']['phase']", resources))
+                break
+            raise AnsibleFilterError(msg("['status']['phase']", resources))
+
+    return filtered
+
+
+def filter_ready(resources, ready=True, skip_missing=True):
```

Comment:
Again, I don't think `ready` should be a parameter here. I don't think a filter called `filter_ready` should get a boolean argument that flips the logic backwards. (Often boolean arguments for a function are a code smell.) I'd rather have `filter_ready` do one thing and one thing only, and return me resources which are in ready status.

Reply:
Agree with `filter_ready` `param: ready` is not needed